app/assets/images/square-gsa-dark.svg

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
i, for one, hate the little trademark mark that makes this not a perfect square -- is it needed?